railties/test/application/test_runner_test.rb

```diff
@@ -536,6 +536,21 @@ def test_rake_passes_multiple_TESTOPTS_to_minitest
       assert_match "seed=1234", output, "passing TEST= should run selected test"
     end
 
+    def test_rake_runs_multiple_test_tasks
+      create_test_file :models, "account"
+      create_test_file :controllers, "accountcontroller"
```

Comment:
Let's follow Rails' naming conventions for good measure: `"accounts_controller"` :)